### ISSUE TITLE
NTR: D-lactate high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8641]
New term request D-lactate high in urine (elthat_190709134346): D-lactic acid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000467
- label: Elevated urine D-lactate level
- parents: [Aciduria (HP:0012072)](https://hpo.jax.org/app/browse/term/HP:0012072)
- pmid: PMID:30931947
- def: The amount of D-lactate in the urine, normalized for urine concentration, is above the upper limit of normal. L-lactate is a familiar molecule to the human body and is also produced in large amounts in human tissues, depending on metabolic conditions. In contrast, D-lactate is produced only in minute quantities in human tissues, and is therefore not detectable in the bloodstream under normal physiological conditions.
- comment: Elevated urine D-lactate may be observed with D-lactic aciduria with susceptibility to gout 	(OMIM:45450).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8641
- synonyms:
